packages/components/src/components/my-position/my-position.tsx

```diff
@@ -0,0 +1,402 @@
+import { Component, Host, JSX, Prop, Event, EventEmitter, State, h } from '@stencil/core';
+import { UAParser } from 'ua-parser-js';
+import merge from 'deepmerge';
+import midtColors from '@mapsindoors/midt/tokens/color.json';
+import midtOpacity from '@mapsindoors/midt/tokens/opacity.json';
+import { getNestedValue } from '../../utils/utils';
+
+enum PositionStateTypes {
+    POSITION_UNKNOWN = 'POSITION_UNKNOWN',
+    POSITION_REQUESTING = 'POSITION_REQUESTING',
+    POSITION_INACCURATE = 'POSITION_INACCURATE',
+    POSITION_KNOWN = 'POSITION_KNOWN',
+    POSITION_CENTERED = 'POSITION_CENTERED',
+    POSITION_TRACKED = 'POSITION_TRACKED',
```

Comment:
What's the difference between known and tracked? 🤔

Reply:
@ammapspeople I couldn't really come up with a better name. 

This is tracked: 
![image](https://github.com/MapsPeople/web-ui/assets/121795063/e4e5fb34-3435-4d1f-ab24-8051d91bb0ea)

This is known:
![image](https://github.com/MapsPeople/web-ui/assets/121795063/0596a8a4-5fea-46db-92e8-964e855c76f4)

Reply:
Ok, that makes more sense.